dotnetv3/dynamodb/FromSQL/CreateTableExample/CreateTableExample/CreateTable.cs

```diff
@@ -1,69 +1,38 @@
-﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX - License - Identifier: Apache - 2.0
-// snippet-start:[dynamodb.dotnet35.CreateTable]
-using System;
-using System.Collections.Generic;
-using System.Configuration;
-using System.Threading.Tasks;
-using Amazon;
-using Amazon.DynamoDBv2;
-using Amazon.DynamoDBv2.Model;
 
 namespace CreateTable
 {
-    public class CreateTable
-    {
-        public static async Task<CreateTableResponse> MakeTableAsync(IAmazonDynamoDB client, string table)
-        {
-            var response = await client.CreateTableAsync(new CreateTableRequest
-            {
-                TableName = table,
-                AttributeDefinitions = new List<AttributeDefinition>
-                {
-                    new AttributeDefinition
-                    {
-                        AttributeName = "ID",
-                        AttributeType = "S"
-                    },
-                    new AttributeDefinition
-                    {
-                        AttributeName = "Area",
-                        AttributeType = "S"
-                    }
-                },
-                KeySchema = new List<KeySchemaElement>
-                {
-                    new KeySchemaElement
-                    {
-                        AttributeName = "ID",
-                        KeyType = "HASH"
-                    },
-                    new KeySchemaElement
-                    {
-                        AttributeName = "Area",
-                        KeyType = "RANGE"
-                    }
-                },
-                ProvisionedThroughput = new ProvisionedThroughput
-                {
-                    ReadCapacityUnits = 10,
-                    WriteCapacityUnits = 5
-                }
-            });
+    using System;
+    using System.Collections.Generic;
+    using System.Configuration;
+    using System.Threading.Tasks;
+    using Amazon;
+    using Amazon.DynamoDBv2;
+    using Amazon.DynamoDBv2.Model;
 
-            return response;
-        }
+    // snippet-start:[dynamodb.dotnet35.CreateTable]
 
-        static void Main()
+    /// <summary>
+    /// Shows how to create an Amazon DynamoDB table. The example was created
+    /// using the AWS SDK for .NET version 3.7 and .NET Core 5.0.
+    /// </summary>
+    public class CreateTable
+    {
+        /// <summary>
+        /// Retrieves configuration information, creates the DynamoDB client,
+        /// and calls MakeTableAsync to create the table.
+        /// </summary>
+        public static async Task Main()
         {
             var configfile = "app.config";
             string region;
             string table;
-                        
+
             // Get default Region and table from config file
```

Comment:
AWS Region

Reply:
Fixed and added period to end of sentence.